sympy/functions/combinatorial/tests/test_comb_factorials.py

```diff
@@ -549,6 +549,53 @@ def test_binomial_rewrite():
     assert binomial(n, x).rewrite(ff) == binomial(n, x)
 
 
+def test_multinomial():
+    from sympy import I
+    n, k = symbols('n k', integer=True)
+    nz = Symbol('nz', integer=True, nonzero=True)
+    kp = Symbol('kp', integer=True, positive=True)
+    z = Symbol('z', zero=True)
+    nt = Symbol('nt', integer=False)
+
+    assert multinomial(0, 0) == 1
+    assert multinomial(1, 1) == 2
+    assert multinomial(1, 1, 1) == 6
+    assert multinomial(1, 2) == 3
+    assert multinomial(-1, 2) == 0
+    assert multinomial(1, -1) == 0
+    assert multinomial(S.Half, S.Half) == 4/pi
+    assert multinomial(n, -1) == 0 # holds for all integers (negative, zero, positive)
+    assert multinomial(kp, -1) == 0
+    assert multinomial(nz, 0) == 1
+    assert multinomial(4321, 51) == 227457163650726634405257688425597243324930864330979686419067341046916573628729411790197413544787445180629754503984973472
```

Comment:
This is not necessary. The gamma function are already tested so symbolic and a few numeric tests are all that are needed.

Reply:
alright ... I'll remove some redundant ones 😄

Reply:
I've only left a couple of numeric variants and then symbolic ones, ones with complex numbers and finally tests for expansions.